tests/ClientTest.php

```diff
@@ -159,19 +159,57 @@ public function testCaptureEventAttachesStacktraceAccordingToAttachStacktraceOpt
             }))
             ->willReturn('500a339f3ab2450b96dee542adf36ba7');
 
-        $client = ClientBuilder::create(['attach_stacktrace' => $shouldAttachStacktrace])
+        $client = ClientBuilder::create(['attach_stacktrace' => $attachConfigSetting])
             ->setTransportFactory($this->createTransportFactory($transport))
             ->getClient();
 
-        $this->assertEquals('500a339f3ab2450b96dee542adf36ba7', $client->captureEvent([]));
+        $this->assertEquals('500a339f3ab2450b96dee542adf36ba7', $client->captureEvent($payload));
     }
 
-    public function captureEventAttachesStacktraceAccordingToAttachStacktraceOptionDataProvider(): array
+    public function captureEventAttachesStacktraceAccordingToAttachStacktraceOptionDataProvider(): \Generator
     {
-        return [
-            [true],
-            [false],
+        yield 'attaches a stacktrace when attach_stacktrace=true but none is provided' => [
```

Comment:
Please change this description to `Stacktrace attached when attach_stacktrace = true and both payload.exception and payload.stacktrace are set`

Reply:
shouldn't this be "… and both payload.exception and payload.stacktrace are **unset**"?

Reply:
Yes, nice catch!

Reply:
fixed